### PR TITLE
Update homepage

### LINKED ISSUE #7403: 24 January homepage content update
For the **24 January** push, please update the homepage per these [programming details](https://docs.google.com/document/d/1g4RuGKW__eTHNd277rKPMM46ygonXzW68jSs2OXEkPM/edit#). 

@muffinresearch @bobsilverberg

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #7403

### CODE REVIEW

File: src/amo/sagas/home.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ export function* fetchHomeAddons({
       page_size: String(LANDING_PAGE_THEME_COUNT),
     },
   };
-  const popularExtensionsParams: SearchParams = {
+  const popularAddonsParams: SearchParams = {
     api: state.api,
     filters: {
       addonType: ADDON_TYPE_EXTENSION,
```
Comment (bobsilverberg) [inline src/amo/sagas/home.js]: I think this is meant to be `getAddonTypeFilter(ADDON_TYPE_THEME)`, in order to return themes and not extensions.

File: src/amo/pages/Home/index.js [hunk 2]
```diff
@@ -53,17 +54,23 @@ export const isFeaturedCollection = (
...
-      header: i18n.gettext('News boosters'),
+      footerText: i18n.gettext('See more YouTube boosters'),
+      header: i18n.gettext('YouTube boosters'),
       isTheme: false,
       ...FEATURED_COLLECTIONS[0],
     },
     {
-      footerText: i18n.gettext('See more weather reports'),
-      header: i18n.gettext('Weather reports'),
+      footerText: i18n.gettext('See more health & wellness add-ons'),
```
Comment (rebmullin) [inline src/amo/pages/Home/index.js]: nit: just wondering...should this say "See more health & wellness **extensions**?

File: src/amo/pages/Home/index.js [hunk 3]
```diff
@@ -53,17 +54,23 @@ export const isFeaturedCollection = (
...
+      header: i18n.gettext('Health & Wellness'),
       isTheme: false,
       ...FEATURED_COLLECTIONS[1],
     },
+    {
+      footerText: i18n.gettext('See more productivity add-ons'),
```
Comment (rebmullin) [inline src/amo/pages/Home/index.js]: same here?

File: src/amo/pages/Home/index.js [hunk 4]
```diff
@@ -341,24 +348,29 @@ export class HomeBase extends React.Component {
...
-          footerText={i18n.gettext('See more popular extensions')}
+          addons={shelves.popularAddons}
+          className="Home-PopularAddons"
+          header={i18n.gettext('Popular themes')}
+          footerText={i18n.gettext('See more popular themes')}
           footerLink={{
             pathname: '/search/',
             query: {
-              addonType: ADDON_TYPE_EXTENSION,
+              q: 'themes',
```
Comment (bobsilverberg) [inline src/amo/pages/Home/index.js]: Do we really want/need `q=themes` here? I know that's in the requirements doc, but I don't understand it. Normally just having the `addonType` is enough. @devaneymoz, can you comment on this?

File: src/amo/pages/Home/index.js (hunk 2 above)
Comment (devaneymoz) [reply src/amo/pages/Home/index.js]: Yes, it should say "extensions".

File: src/amo/pages/Home/index.js (hunk 3 above)
Comment (devaneymoz) [reply src/amo/pages/Home/index.js]: "See more productivity extensions"

File: src/amo/sagas/home.js (hunk 1 above)
Comment (devaneymoz) [reply src/amo/sagas/home.js]: @bobsilverberg Sorry I'm confused where this appears? Where are we attempting to direct users to popular themes?

File: src/amo/pages/Home/index.js (hunk 4 above)
Comment (devaneymoz) [reply src/amo/pages/Home/index.js]: @bobsilverberg Sorry but again I'm unclear where precisely we're looking at on the page?

Comment (bobsilverberg) [reply src/amo/pages/Home/index.js]: @devaneymoz It's the link in the header of the shelf where we list popular themes:

screenshot 2019-01-16 15 43 26

**Comment by devaneymoz** [comment]
@bobsilverberg It should display as it does in your screen grab: "See more popular themes"

File: src/amo/pages/Home/index.js (hunk 2 above)
Comment (willdurand) [reply src/amo/pages/Home/index.js]: :+1:

File: src/amo/pages/Home/index.js (hunk 3 above)
Comment (willdurand) [reply src/amo/pages/Home/index.js]: :+1:

File: src/amo/pages/Home/index.js (hunk 4 above)
Comment (willdurand) [reply src/amo/pages/Home/index.js]: > Do we really want/need `q=themes` here?

I don't know.. That was in the Google doc, but maybe not. I removed it for now, we can still add it later.

**Pull request merged**